lms/djangoapps/courseware/module_render.py

```diff
@@ -191,12 +191,14 @@ def toc_for_course(user, request, course, active_chapter, active_section, field_
                     }
 
                     #
-                    # Add in rendering context for proctored exams
+                    # Add in rendering context for proctored exams and timed exams
                     # if applicable
                     #
                     is_proctored_enabled = (
-                        getattr(section, 'is_proctored_enabled', False) and
-                        settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False)
+                        (
+                            getattr(section, 'is_proctored_enabled', False) or
+                            getattr(section, 'is_time_limited', False)
+                        ) and settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False)
```

Comment:
Hrm, actually as I think about it some more, we're going to hit the same problem writing unit tests because we are dependent on a version of edx-proctoring greater than 0.8.3

I'm going to have to think about this a bit more.